src/olympia/lib/settings_base.py

```diff
@@ -87,37 +87,29 @@ def path(*folders):
 ABUSE_EMAIL = 'amo-admins+ivebeenabused@mozilla.org'
 NOBODY_EMAIL = 'nobody@mozilla.org'
 
+DRF_API_VERSIONS = ['v3', 'v4']
+DRF_API_REGEX = r'^/?api/(?:v3|v4)/'
+
 # Add Access-Control-Allow-Origin: * header for the new API with
 # django-cors-headers.
 CORS_ORIGIN_ALLOW_ALL = True
-CORS_URLS_REGEX = r'^/api/v3/.*$'
+CORS_URLS_REGEX = DRF_API_REGEX
 
 
 # Sadly the term WHITELIST is used by the library
 # https://pypi.python.org/pypi/django-cors-headers-multi/1.2.0
-# TODO(andym): see if I can get them to accept a patch for that.
-def cors_endpoint_overrides(internal, public):
+def cors_endpoint_overrides(list_):
```

Comment:
I think `list_` can be renamed to `whitelisted_endpoints`?

Reply:
https://github.com/mozilla/addons-server/pull/8291/commits/8d43518f9b3956e194ae42af1bed322422f0ec1d